app/styles/app.scss

```diff
@@ -2,11 +2,6 @@
 // VENDORS
 //
 
-// Normalize.css
-// A modern, HTML5-ready alternative to CSS resets
-// https://necolas.github.io/normalize.css/
-@import "normalize";
-
```

Comment:
normalize is imported straight into `vendor.css` from the `normalize.css` vendor package